tests/mq/plugins/test_broFixup.py

```diff
@@ -0,0 +1,1697 @@
+            "last_reply":"250 2.0.0 Ok: queued as 3E1EC13F655",
+            "path":["128.199.139.6","63.245.214.155","127.0.0.1","10.22.82.42"],
+            "tls":'false',
+            "fuids":["FnR86s3vp0xKw286Ei","FiYNQo4ygv3xPAeocd"],
+            "is_webmail":'false',
+            'category': 'bro',
+            'type': 'smtp',
+            'customendpoint': ' '
+        }
+        result, metadata = self.plugin.onMessage(event, self.metadata)
+        self.verify_defaults(result)
+        self.verify_metadata(metadata)
+        assert toUTC(event['ts']).isoformat() == result['utctimestamp']
+        assert toUTC(event['ts']).isoformat() == result['timestamp']
+        assert sorted(result['details'].keys()) == sorted(event.keys())
+        #assert result['summary'] == 'SMTP: 63.245.214.155 -> 128.199.139.6:25 from \u0022Bugzilla@Mozilla\u0022 <bugzilla-daemon@mozilla.org> to bugmail@firebot.glob.uno ID <bug-1400759-507647@https.bugzilla.mozilla.org/>'.decode('unicode-escape')
+    
+    def test_smtp_log2(self):
+        event = {
+            "ts":1505703597.295432,
+            "uid":"Ct7e4waRBwsLoRvfg",
+            "sourceipaddress":"63.245.214.155",
+            "sourceport":4523,
+            "destinationipaddress":"128.199.139.6",
+            "destinationport":25,
+            "trans_depth":1,
+            "helo":"smtp.mozilla.org",
+            "mailfrom":"bugzilla-daemon@mozilla.org",
+            "rcptto":["bugmail@firebot.glob.uno"],
+            "date":"Mon, 18 Sep 2017 02:59:56 +0000",
+            "subject":"[Bug 1400759] New: Debugger script search not working when content type = \u0027image/svg+xml\u0027",
+            "first_received":"by jobqueue2.bugs.scl3.mozilla.com (Postfix, from userid 0)\u0009id 87345380596; Mon, 18 Sep 2017 02:59:56 +0000 (UTC)",
+            "second_received":"from jobqueue2.bugs.scl3.mozilla.com (jobqueue2.bugs.scl3.mozilla.com [10.22.82.42])\u0009by mx1.mail.scl3.mozilla.com (Postfix) with ESMTPS id 9EBCBC0A97\u0009for <bugmail@firebot.glob.uno>; Mon, 18 Sep 2017 02:59:56 +0000 (UTC)",
+            "last_reply":"250 2.0.0 Ok: queued as 3E1EC13F655",
+            "path":["128.199.139.6","63.245.214.155","127.0.0.1","10.22.82.42"],
+            "tls":'false',
+            "fuids":["FnR86s3vp0xKw286Ei","FiYNQo4ygv3xPAeocd"],
+            "is_webmail":'false',
+            'category': 'bro',
+            'type': 'smtp',
+            'customendpoint': ' '
+        }
+
+        result, metadata = self.plugin.onMessage(event, self.metadata)
+        self.verify_defaults(result)
+        self.verify_metadata(metadata)
+        assert toUTC(event['ts']).isoformat() == result['utctimestamp']
+        assert toUTC(event['ts']).isoformat() == result['timestamp']
+        assert sorted(result['details'].keys()) == sorted(event.keys())
+        assert 'from' in result['details']
+        assert 'to' in result['details']
+        assert 'msg_id' in result['details']
```

Comment:
We should verify the summary is set correctly here

Reply:
Done